pkg/harvester/edit/harvesterhci.io.storage/index.vue

```diff
@@ -128,6 +142,16 @@ export default {
   watch: {
     provisionerWatch() {
       this.$set(this.value, 'parameters', {});
+    },
+
+    allowedTopologies(neu) {
+      if (!neu || neu.length === 0) {
+        delete this.value.allowedTopologies;
+
+        return;
+      }
+
+      this.value.allowedTopologies = [{ matchLabelExpressions: neu }];
     }

```

Comment:
![image](https://user-images.githubusercontent.com/24985926/232669142-3681316b-bb78-4412-a37c-9bf0f5a83f14.png)

A small enhancement: When entering a value for "allowedTopologies" and then deleting it, there may be an issue when saving.

Reply:
done